incubator/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/config/SchemaRegistryOptionsConfigAdapter.java

```diff
@@ -78,6 +92,15 @@ public OptionsConfig adaptFromJson(
             {
                 options.context(object.getString(CONTEXT));
             }
+
+            if (object.containsKey(MAX_AGE_NAME))
+            {
+                options.maxAge(Duration.ofSeconds(object.getJsonNumber(MAX_AGE_NAME).longValue()));
+            }
+            else
+            {
+                options.maxAge(Duration.ofSeconds(300));
+            }
```

Comment:
Handle defaults via Constantine type Duration instead of hardcoding.
Also defaulting should be handled in the builder and the builder is called conditionally from the config adapter so value defaults if not present.